test/linalg/cholesky.jl

```diff
@@ -94,18 +91,12 @@ using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted, PosDefException
         U = Bidiagonal([2,sqrt(eltya(3))],[-1],true) / sqrt(eltya(2))
         @test full(chol(S)) ≈ full(U)
 
-        #lower Cholesky factor
```

Comment:
My suggestion to preserve/revise the comment associated with that block was independent of whether that comment applies primarily or strictly to the deprecated tests: There is no other comment describing the surviving contents of that block, and having an up to date comment describing the remaining tests in that block would be worthwhile :).